sdk/core/core-http/review/core-http.api.md

```diff
@@ -49,6 +49,24 @@ export class ApiKeyCredentials implements ServiceClientCredentials {
 // @public
 export function applyMixins(targetCtor: any, sourceCtors: any[]): void;
 
+// @public
+export class AuthenticationChallenge {
+    constructor(authorization: string, scope: string);
+    // (undocumented)
```

Comment:
Interesting, so:

In Key Vault, these functionalities are never published. Now we should probably add documentation.

Reply:
If not in this PR, perhaps we could make an issue.

Reply:
I'd prefer it be in this PR, since these types are made public in this PR